CH06_AudioConverter/CH06_AudioConverter/main.c

```diff
@@ -68,7 +68,9 @@ OSStatus MyAudioConverterCallback(AudioConverterRef inAudioConverter,
 								  AudioStreamPacketDescription **outDataPacketDescription,
 								  void *inUserData)
 {
-	MyAudioConverterSettings *audioConverterSettings = (MyAudioConverterSettings *)inUserData;
+void *sourceBuffer = 0;
```

Comment:
C convention is to use `NULL` for uninitialized pointers, not `0` (though i can't promise we get it right throughout the book).